src/main/java/com/github/calvincodes/github/SearchableLabels.java

```diff
@@ -7,11 +7,10 @@
 public class SearchableLabels {
 
     public static List<String> SEARCHABLE_LABELS = new ArrayList<>(
-            Arrays.asList(
-                    "newbie",
-                    "first-timers-only",
-                    "\"good first issue\"", // Official GitHub issue.
-                    "good-first-issue",
-                    "beginner"
+            Arrays.asList("beginner",
+            "first-timers-only",
+            "good-first-issue",
+            "\"good first issue\"", // Official GitHub issue.
+            "newbie"
```

Comment:
The order looks fine, but can you maintain the same indentation and formatting?
Start the list elements with a new line after `Arrays.asList(`, and provide same spacing as in the master.
Overall, this will look something like:
```
            Arrays.asList(
                    "beginner",
                    "first-timers-only",
                    "good-first-issue",
                    "\"good first issue\"", // Official GitHub issue.
                    "newbie"
            ));
```